### DISCUSSION TITLE
Unable to enforce HTTP with a custom domain - github pages.

### DISCUSSION THREAD

**Discussion by itsvalentinaparra** [community/community#48491] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi! 
Hope you are doing well!

I'm sending this discussion because I'm having a problem with selecting a custom domain, I attach a screen shot of the problem.

![Captura de Pantalla 2023-02-24 a la(s) 11 31 25](https://user-images.githubusercontent.com/111432898/221203878-08e37fa5-9d0f-447e-8f0e-99300fb1734f.png)

In addition to this, this is what it's placed on the domain page:
![Github question](https://user-images.githubusercontent.com/111432898/221204079-8ca95171-d565-4aee-9744-17eda947bebb.png)

I've been researching a lot about it (even on other similar discussions) and I haven't be able to fix it, I'd appreciate some help. 

Thanks in advance! :)

**Comment by yoannchaudet** [comment]
Remove the IP addresses which are not GitHub Pages in your `A` records. We do check for that.

These especially:

```
dylanmolloy.com.	600	IN	A	3.33.152.147
dylanmolloy.com.	600	IN	A	15.197.142.173
dylanmolloy.com.	600	IN	A	97.74.100.11
dylanmolloy.com.	600	IN	A	173.201.68.11
```

**Reply by itsvalentinaparra** [reply]
Now it works. Thank you for you answer!
